certbot/tests/main_test.py

```diff
@@ -1164,6 +1165,69 @@ def test_update_registration_with_email(self, mock_utility, mock_email):
                             email in mock_utility().add_message.call_args[0][0])
 
 
+class UnregisterTest(unittest.TestCase):
+    def setUp(self):
+        self.patchers = {
```

Comment:
I don't like it either, but the alternative was dumping these tests in `TestMain`.